### DISCUSSION TITLE
Attached stems and sugarcanes are break everytime get a block update?

### DISCUSSION THREAD

**Discussion by Ouranos-dev** [FabricMC/fabric-api#2866] [Player support]
Like im said in title.

**Comment by lolouthefox** [comment]
Can you give more informations about this issue? Like a GIF, a picture or a video so we can see what is happening and help you. Also give a list of the downloaded FabricMC mods.

**Reply by Ouranos-dev** [reply]
https://user-images.githubusercontent.com/36356983/214917468-fcfef8f9-ef86-4a89-b9fa-2eec45a884b3.mp4
[modlist.txt](https://github.com/FabricMC/fabric/files/10512253/modlist.txt)
